cmd/testpeers.go

```diff
@@ -45,12 +47,16 @@ type testPeersConfig struct {
 }
 
 type testCasePeer func(context.Context, *testPeersConfig, host.Host, p2p.Peer) testResult
+type testCasePeerSelf func(context.Context, *testPeersConfig) testResult
+type testCaseRelay func(context.Context, *testPeersConfig, string) testResult
 
 const (
 	thresholdPeersMeasureAvg = 50 * time.Millisecond
 	thresholdPeersMeasureBad = 240 * time.Millisecond
 	thresholdPeersLoadAvg    = 50 * time.Millisecond
 	thresholdPeersLoadBad    = 240 * time.Millisecond
+	thresholdRelayMeasureAvg = 50 * time.Millisecond
+	thresholdRelayMeasureBad = 240 * time.Millisecond
```

Comment:
@LukeHackett12 can I get your opinion on those 2 values? Those are the thresholds after which we consider average or bad performance for a RTT to a charon relay.